eurito_daps/packages/utils/test_db.py

```diff
@@ -0,0 +1,71 @@
+'''A utility script, which outputs records from the specified tables in the database

```

Comment:
What's this script for? It looks like it's for testing, but the ORM is in here??

Reply:
removed